install/runtime-payload/Makefile

```diff
@@ -1,5 +1,14 @@
-binaries: 
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+.PHONY: binaries build
+
+.PHONY: binaries
+binaries:
 	hack/binaries.sh
 
+.PHONY: image
```

Comment:
Should the target `build` be changed to `image`?

Reply:
DaLi is on vacation for Spring Festival, so I've pushed https://github.com/confidential-containers/cloud-api-adaptor/pull/521/commits/3d86db07e6d122e3aefc24d76beff5700eceaad1 to cover this update. Hopefully I've understood the request :)

Reply:
yeah the changes looks good :-)